corehq/apps/callcenter/tasks.py

```diff
@@ -46,6 +47,17 @@ def calculate_indicators():
         indicator_set.get_data()
 
 
+def sync_web_user_usercases_if_applicable(user, domain, spawn_task):
+    domain_obj = Domain.get_by_name(domain)
+    if not domain_obj and settings.UNIT_TESTING:
```

Comment:
Why bother checking for `domain_obj` here?

Reply:
I was being overly safe. For some reason if `Domain.get_by_name` returned None for some inexplicable reason this would catch it. In hindsight this is probably not necessary.

Reply:
Nah, you're right, it's better to be cautious.